config.go

```diff
@@ -63,6 +63,12 @@ type BurrowConfig struct {
 		ZookeeperPort int      `gcfg:"zookeeper-port"`
 		ZookeeperPath []string `gcfg:"zookeeper-path"`
 	}
+	Druid map[string]*struct {
+		Zookeepers    []string `gcfg:"zookeeper"`
```

Comment:
Do we actually do anything with zookeeper for Druid?  Code seems to indicate we make the connection but do nothing with it.